SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -274,12 +278,16 @@ FLevelData GenerateSchemaForSublevel(UWorld* World)
 
 void GenerateSchemaForSublevels(const FString& SchemaPath)
 {
-	UWorld* EditorWorld = GEditor->GetEditorWorldContext().World();
+	if (GWorld == nullptr)
+	{
+		UE_LOG(LogTemp, Log, TEXT("No world found, skipping schema for sublevels."));
```

Comment:
When does this happen? Can you put a bit of context in comments?

Reply:
This happened when running the Commandlet since there is no EditorContext.